docs/source/developer_guides/model_merging.md

```diff
@@ -138,3 +138,14 @@ print(tokenizer.decode(outputs[0]))
 
 </hfoption>
 </hfoptions>
+
+
+## Merging $(IA)^3$ Models
```

Comment:
```suggestion
## Merging (IA)³ Models
```

LaTeX syntax doesn't seem to be supported: https://moon-ci-docs.huggingface.co/docs/peft/pr_1701/en/developer_guides/model_merging#merging-ia3-models